examples/foomedical/src/test.setup.ts

```diff
@@ -1,7 +1,11 @@
 import '@testing-library/jest-dom';
+import { TextDecoder, TextEncoder } from 'node:util';
 
 (globalThis as any).IS_REACT_ACT_ENVIRONMENT = true;
 
+Object.defineProperty(globalThis.window, 'TextDecoder', { value: TextDecoder });
```

Comment:
Curious why these changes were needed.

Reply:
`<BrowserRouter>` requires it, not sure why? 🤷‍♂️